src/GNNGraphs/temporalsnapshotsgnngraph.jl

```diff
@@ -239,4 +239,14 @@ function print_feature_t(io::IO, feature)
     else 
         print(io, "no")
     end
+end
+
+# move temporal graph to gpu
+function Flux.gpu(tsg::TemporalSnapshotsGNNGraph)
+    return TemporalSnapshotsGNNGraph(
+        Flux.gpu(tsg.num_nodes),
+        Flux.gpu(tsg.num_edges),
+        tsg.num_snapshots,
+        Flux.gpu(tsg.snapshots),
+        Flux.gpu(tsg.tgdata))
 end
```

Comment:
I think we can avoid implementing gpu by just defining `@functor TemporalSnapshotsGNNGraph`